pkgs/applications/window-managers/phosh/phosh-mobile-settings.nix

```diff
@@ -17,16 +17,20 @@
 , wayland-protocols
 , json-glib
 , gsound
+, gmobile
 }:
 
 stdenv.mkDerivation rec {
   pname = "phosh-mobile-settings";
-  version = "0.39.0";
+  version = "0.41.0";
 
-  src = fetchurl {
-    # This tarball includes the meson wrapped subproject 'gmobile'.
-    url = "https://sources.phosh.mobi/releases/${pname}/${pname}-${version}.tar.xz";
-    hash = "sha256-9vN4IqGoRHDJQYohycrrSj4ITJHHaSNgPjpEjRCCvUw=";
+  src = fetchFromGitLab {
```

Comment:
Why switch? the `updateScript` are still `directoryListingUpdater`

Reply:
IIRC I did this because of [`build: Allow to use shared gmobile`](https://gitlab.gnome.org/World/Phosh/phosh-mobile-settings/-/commit/70dd6e7111ca43c5d533b787e1accdd550fdcb95). Forgot to check the updateScript